Authenticator/Source/TableDiff.swift

```diff
@@ -74,29 +94,29 @@ private func changes<Row>(from oldRows: ArraySlice<Row>, to newRows: ArraySlice<
 }
 
 private func changesWithInsertion<Row>(from oldRows: ArraySlice<Row>, to newRows: ArraySlice<Row>,
-    comparator isSameRow: (Row, Row) -> Bool) -> [Change]
+    comparator: (Row, Row) -> Bool, equation: (Row, Row) -> Bool) -> [Change]
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)